src/src/fileLoger/hillel/FileLogger.java

```diff
@@ -0,0 +1,65 @@
+package fileLoger.hillel;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class FileLogger {
+    private FileLoggerConfiguration config;
+
+    public FileLogger(String fileName) throws IOException {
```

Comment:
Would be more flexible to have 2 constructors: the first accepts 'FileLoggerConfiguration` and just assigns it and the second one is a default constructor that accepts no parameters but calls `FileLoggerConfigurationLoader.load(fileName)` to assign `config`